src/coreclr/src/gc/gcinterface.h

```diff
@@ -942,6 +942,7 @@ struct ScanContext
     uintptr_t stack_limit; // Lowest point on the thread stack that the scanning logic is permitted to read
     bool promotion; //TRUE: Promotion, FALSE: Relocation.
     bool concurrent; //TRUE: concurrent scanning
+    bool scanStatics; //TRUE: scan statics, not stack
```

Comment:
The contract for this looks odd: The GC can set it, but then the EE side can decide that it is not a good idea to scan the statics and ignore it. 

Should this rather be an argument for `GCToEEInterface::GcScanRoots` and have the policy to decide whether or not to scan statics on the GC side? Or maybe even have a new `GCToEEInterface::GcScanStatics` method on GC/EE interface?